articles/tutorials/create-a-sample-app/02-create-a-petclinic-csharp/create-a-petclinic-csharp.md

```diff
@@ -332,6 +332,10 @@ It should also have a success result as well as an array of JSON results includi
 
 If you wish to continue exploring the concepts covered by this tutorial you can make new services, operations and DTOs to cover additional use cases towards a complete implementation of the PetClinic.
 
+### Get the completed solution
+
+Alternatively you can view the fully fledged solution for multiple languages on [github](https://github.com/IntentSoftware/Samples-PetClinic).
```

Comment:
Couple of minor comments:

- When I saw "multiple languages", for a moment I was thinking of spoken/written languages like "English" and "Afrikaans".
- Pretty sure Github should always have a capital G.

`We have a complete example of the Pet Clinic application implemented in multiple coding languages on [Github](https://github.com/IntentSoftware/Samples-PetClinic).`